### PR TITLE
feat: new ictinus v5 migration

### PR DESCRIPTION
**Pull request by panvourtsis**
## Description

This PR introduce new v5 ictinus and all changes are made by running the codemod. Tests are expecting to work after #157merged to this

* https://github.com/Orfium/toolbox/pull/157 fixing tests of this

### CODE REVIEW

File: src/ui/TopBar/components/FancyUserMenu/UserMenu.styles.ts [hunk 1]
```diff
@@ -52,7 +51,8 @@ export const MenuOuterWrapper = styled.div`
     cursor: pointer;
 
     &:focus-visible {
-      outline: ${({ theme }) => getFocus({ theme }).styleOutline};
+      // @TODO fix this
```
[the remark this answers is not in the document]
Comment (panvourtsis) [reply src/ui/TopBar/components/FancyUserMenu/UserMenu.styles.ts:54]: @SVonKarajann is working on another PR for it

File: src/ui/TopBar/components/FancyUserMenu/UserMenu.styles.ts [hunk 2]
```diff
@@ -163,7 +163,8 @@ const menuItemStyles = (theme: Theme) => css`
   }
 
   &:focus-visible {
-    outline: ${getFocus({ theme }).styleOutline};
+    // @TODO fix this
```
[the remark this answers is not in the document]
Comment (panvourtsis) [reply src/ui/TopBar/components/FancyUserMenu/UserMenu.styles.ts:166]: @SVonKarajann is working on another PR for it

File: src/ui/TopBar/components/UserMenu/UserMenu.tsx [hunk 3]
```diff
@@ -31,84 +24,44 @@ function UserMenu(props: UserMenuProps) {
...
-  }, []);
-
-  useClickAwayListener(outerMenuWrapperRef, (e: MouseEvent) => {
-    if (open) {
-      e.stopPropagation();
-      setOpen(false);
-    }
-  });
+  const btnRef = useRef(null);
+  const handleBtnClick = (e: any) => {
```
[the remark this answers is not in the document]
Comment (panvourtsis) [reply src/ui/TopBar/components/UserMenu/UserMenu.tsx]: resolved at https://github.com/Orfium/toolbox/pull/156/commits/5c6da88f6911b26076321c2164e16c0c9a7792e6

File: src/ui/Navigation/components/OrganizationSelector/OrganizationSelector.tsx [hunk 4]
```diff
@@ -39,29 +37,27 @@ function OrganizationSelector(props: Props & TestProps) {
     disabled,
     onSelect,
     buttonText = 'More',
-    menuPosition = 'left',
     dataTestId,
     tagText = 'Organization',
   } = props;
   const [open, setOpen] = useState(false);
+  const btnRef = useRef(null);
+  const handleBtnClick = (e: any) => {
```
[the remark this answers is not in the document]
Comment (panvourtsis) [reply src/ui/Navigation/components/OrganizationSelector/OrganizationSelector.tsx]: resolved at https://github.com/Orfium/toolbox/pull/156/commits/5c6da88f6911b26076321c2164e16c0c9a7792e6

**Review by tsevdos — APPROVED** [review]

**Pull request merged**